src/main/scala/org/allenai/plugins/CoreRepositories.scala

```diff
@@ -28,8 +24,7 @@ object CoreRepositories {
     val defaults = Seq(
       spray,
       Resolver.jcenterRepo,
-      typesafeReleases,
-      ai2PublicReleases
```

Comment:
I can leave this in (which auto-adds Bintray), but I think it might be better to just require projects to do this explicitly.